pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -313,13 +305,16 @@ def on_evaluation_batch_start(self, batch, batch_idx, dataloader_idx):
         else:
             self.trainer.call_hook('on_validation_batch_start', batch, batch_idx, dataloader_idx)
 
-    def on_evaluation_batch_end(self, *args, **kwargs):
+    def on_evaluation_batch_end(self, output, batch, batch_idx, dataloader_idx):
```

Comment:
Replacing *args **kwargs with explicit variables is fine? I'm not sure myself.

Reply:
Yes, it is :)

Reply:
In trainer.py, we call evaluation_loop. on_evaluation_batch_end(output, batch, batch_idx, dataloader_idx), so yes, everything fine :)